pkg/installer/installer.go

```diff
@@ -45,27 +53,36 @@ const (
 	UninstallSkipEtcdConfig       = "spec.uninstall.skipEtcd"
 
 	// dir and file names for in memory fs
-	etcdDir           = "etcd"
-	stosDir           = "storageos"
-	operatorDir       = "operator"
-	clusterDir        = "cluster"
-	stosOperatorFile  = "storageos-operator.yaml"
-	stosClusterFile   = "storageos-cluster.yaml"
-	etcdOperatorFile  = "etcd-operator.yaml"
-	etcdClusterFile   = "etcd-cluster.yaml"
-	kustomizationFile = "kustomization.yaml"
+	etcdDir              = "etcd"
+	stosDir              = "storageos"
+	operatorDir          = "operator"
+	clusterDir           = "cluster"
+	stosOperatorFile     = "storageos-operator.yaml"
+	stosClusterFile      = "storageos-cluster.yaml"
+	stosSecretsFile      = "storageos-secrets.yaml"
+	csiSecretsFile       = "storageos-csi-secrets.yaml"
+	stosStorageClassFile = "storageos-storageclass.yaml"
+	etcdOperatorFile     = "etcd-operator.yaml"
+	etcdClusterFile      = "etcd-cluster.yaml"
+	kustomizationFile    = "kustomization.yaml"
+	kubeDir              = "/root/.kube"
+	uninstallDir         = "uninstall-"
 
 	// kustomization template
 	kustTemp = `apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:`
 
+	// other defaults
 	stosClusterKind        = "StorageOSCluster"
 	etcdClusterKind        = "EtcdCluster"
 	defaultEtcdClusterName = "storageos-etcd"
 	defaultEtcdClusterNS   = "storageos-etcd"
 	stosFinalizer          = "storageos.com/finalizer"
+	storageClassV1         = "fast"
+	// TODO: change from 'storageos-sc' to 'storageos' once change is made
+	storageClassV2 = "storageos-sc"
```

Comment:
This kind of hard coded values make installer in-flexible. Is it possible to determine storage class from manifests?

Reply:
Yes, good point - we can get this from `Spec.StorageClassName`